package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "todolist",
+  "version": "1.0.0",
+  "description": "A simple list of To Do tasks",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/craigs40/ToDoList.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/craigs40/ToDoList/issues"
+  },
+  "homepage": "https://github.com/craigs40/ToDoList#readme",
+  "dependencies": {
+    "lodash": "^4.17.21"
+  },
+  "devDependencies": {
+    "css-loader": "^6.6.0",
+    "html-webpack-plugin": "^5.5.0",
+    "webpack": "^5.69.1",
+    "webpack-cli": "^4.9.2"
+  },
+  "browserslist": [
+    "defaults",
+    "not IE 11"
+  ]
+}
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/155026151-c19cf02e-e8ef-4354-9697-3057d9af5718.png)

Therefore:

- [ ] Please, install and configure the webpack dev server.  You learned how to do it in the previous exercise.  The result in this file should be:

1. After `npm install webpack-dev-server --save-dev`, it shows up in the dependencies.
2. In the "scripts" section, npm start is configured to start the project in development mode.

Example "scripts" section:

```JSON
  "scripts": {
    
    "test": "echo \"Error: no test specified\" && exit 1",
    "build": "webpack",
    "start": "webpack serve --open",
  }
```


That way, we will meet the requirement, and you will have a lot  easier time developing javascript! try it! 💪